web-portal/frontend/utils/Web3Provider.tsx

```diff
@@ -17,39 +17,44 @@ Object.entries(envVariables).forEach(([key, value]) => {
   }
 });
 
-const metadata = {
-  name: "Porters RPC Gateway",
-  description: "Porters RPC Gateway",
-  url: String(NEXT_PUBLIC_APP_URL),
-  icons: ["https://staging.porters.xyz/favicon.ico"],
-};
-
 // Create wagmiConfig
 export const chains = [mainnet, optimism, base, gnosis] as const;
 
-export const config = defaultWagmiConfig({
-  chains,
-  projectId,
-  metadata,
-  ssr: true,
-  storage: createStorage({
-    storage: cookieStorage,
-  }),
-});
+export const config = createConfig(
+  getDefaultConfig({
+    chains: [mainnet, base, gnosis, optimism],
+    transports: {
+      [mainnet.id]: http(
+          `https://eth-mainnet.rpc.grove.city/v1/1eec3963`
+      ),
+      [base.id]: http(`https://base-mainnet.rpc.grove.city/v1/1eec3963`),
+      [gnosis.id]: http(`https://gnosischain-mainnet.rpc.grove.city/v1/1eec3963`),
+      [optimism.id]: http(`https://optimism-mainnet.rpc.grove.city/v1/1eec3963`),
```

Comment:
Do you mean to be committing these?

Reply:
no regrets! will be disabling!

Reply:
we need to swap to own endpoints at some point